src/mod/irc.mod/irc.c

```diff
@@ -1045,13 +1061,59 @@ static void flush_modes()
   }
 }
 
+static void tell_account_tracking_status(int idx, int details)
+{
+  struct capability *current;
+  int extjoin = 0, notify = 0, tag = 0, whox = use_354;
+  /* List status of account tracking. For 100% accuracy, this requires
+   * WHOX ability (354 messages) and the extended-join and account-notify
+   * capabilities to be enabled.
+   */
+  /* Check if CAPs are enabled */
+  current = cap;
+  while (current != NULL) {
+    if (!strcasecmp("extended-join", current->name) && current->enabled) {
+      extjoin = 1;
+    } else if (!strcasecmp("account-notify", current->name) && current->enabled) {
+      notify = 1;
+    } else if (!strcasecmp("account-tag", current->name) && current->enabled) {
+      tag = 1;
+    }
+    current = current->next;
+  }
+
+  if (whox && notify && extjoin) {
+    dprintf(idx, "%s", "    Account tracking: Perfect\n");
```

Comment:
Can we go back to "enabled" or some other word here? Supported? Fully enabled? I know we've been using this word to describe the scenario, but I think the intended meaning without additional context would be lost on a novice user.

Reply:
perfect idea